src/locale/locales/ja/messages.po

```diff
@@ -170,15 +170,15 @@ msgstr "および"
 
 #: src/view/screens/LanguageSettings.tsx:95
 msgid "App Language"
-msgstr "アプリケーション言語"
+msgstr "アプリの言語"
 
 #: src/view/screens/Settings.tsx:589
 msgid "App passwords"
-msgstr "アプリパスワード"
+msgstr "アプリのパスワード"
```

Comment:
I think it would be better to use the special term "アプリパスワード" rather than "アプリのパスワード". 
I think "アプリのパスワード" may mislead a user that "アプリ" means the official Bluesky app here, so I went with "アプリパスワード".

Reply:
なるほど。